ui/apps/platform/cypress/integration/compliance/Compliance.helpers.js

```diff
@@ -121,7 +119,11 @@ function scanCompliance() {
         { timeout: 20000 }
     );
 
-    cy.get(scanButton).should('not.have.attr', 'disabled');
+    // Note: scan results are polled for every 10 seconds, this should give us plenty of time
+    // to await completion of the scan.
+    cy.get('div:contains("Compliance scanning complete")', {
+        timeout: 30000,
+    });
```

Comment:
Since the scans are done async, we cannot wait on a single request to determine whether or not the scans are complete. Instead of using the `interact/wait` pattern here, this check waits for the UI indication that the scans are finished.